components/eamxx/src/physics/share/tests/physics_saturation_run_and_cmp.cpp

```diff
@@ -189,6 +189,7 @@ struct UnitWrap::UnitTest<D>::TestSaturation
     Scalar mix_ice_mkr;
     Scalar mix_liq_mkr;
 
+    KOKKOS_INLINE_FUNCTION
```

Comment:
@ambrad @bartgol -- could this be a nonbfb change? i am not seeing direct connection between this pr and saturation code (there may be, but i do not see anything obvious), except for this line. Fro Jenkins output
```
[onguba@weaver11 5246]$ grep "saturation.baseline" JENKINS_2024-01-18_124134 | grep "at tol"
Comparing with /home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_sp_debug/data/physics_saturation.baseline4 at tol 1.2e-07
Comparing with /home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/release/data/physics_saturation.baseline8 at tol 2.2e-12
Comparing with /home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_debug/data/physics_saturation.baseline8 at tol 2.2e-16
Comparing with /home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_sp_debug/data/physics_saturation.baseline4 at tol 1.2e-07
Comparing with /home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/release/data/physics_saturation.baseline8 at tol 2.2e-12
Comparing with /home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_debug/data/physics_saturation.baseline8 at tol 2.2e-16
```
So errors maybe not too small for SP runs.

Reply:
Anything's possible, but in principle it shouldn't change answers. But it also seems like it should have not have built without it. My only recommendation is to study this test on Weaver, using `printf`s, etc. One thing that might provide clues is looking at why KOKKOS_INLINE_FUNCTION was apparently not needed on Weaver previously.